RecordingApp/app/src/main/java/wycliffeassociates/recordingapp/ProjectManager/activities/ActivityProjectManager.java

```diff
@@ -95,62 +110,42 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
     //twice will result in two background processes trying to sync the database, and only one reference
     //will be kept in the activity- thus leaking the reference to the first dialog causing in it never closing
     @Override
-    protected void onStart(){
+    protected void onStart() {
         super.onStart();
         //Moved this section to onResume so that these dialogs pop up above the dialog info fragment
         //check if fragment was retained from a screen rotation
         FragmentManager fm = getFragmentManager();
         mExportTaskFragment = (ExportTaskFragment) fm.findFragmentByTag(TAG_EXPORT_TASK_FRAGMENT);
-        mDatabaseResyncTaskFragment = (DatabaseResyncTaskFragment) fm.findFragmentByTag(TAG_DATABASE_RESYNC_FRAGMENT);
-        if(mExportTaskFragment == null){
+        mTaskFragment = (TaskFragment) fm.findFragmentByTag(TAG_TASK_FRAGMENT);
+        //TODO: refactor export to fit the new taskfragment
+        if (mExportTaskFragment == null) {
             mExportTaskFragment = new ExportTaskFragment();
             fm.beginTransaction().add(mExportTaskFragment, TAG_EXPORT_TASK_FRAGMENT).commit();
             fm.executePendingTransactions();
         } else {
-            if(mZipping){
+            if (mZipping) {
```

Comment:
Technically yes, but I felt this was better for readability. Essentially it is if (fragment is null) then { activity was created for the first time } else {recover from orientation change }. So the if (zipping) else if (exporting) could have justification for a private member function (but currently the code here is small, so possibly hurts readability due to the redirection)

All in all though, this code will be deleted following #642